src/newrelic/integration/test.go

```diff
@@ -63,6 +63,7 @@ type Test struct {
 
 	// Remaining fields are populated after the test is run.
 	Skipped bool
+    Warned bool
```

Comment:
```suggestion
        Warned bool
```

Reply:
I think my suggestion probably will not do what we want (tabs vs spaces etc), but this line is misaligned

Reply:
I think [07f7c3ca](https://github.com/newrelic/newrelic-php-agent/pull/642/commits/07f7c3cab645a6d79e453fa2d478eac7fa2a4731) fixes the spacing?